pkg/controllers/spannerautoscaler_controller.go

```diff
@@ -288,6 +288,14 @@ func (r *SpannerAutoscalerReconciler) needUpdateNodes(sa *spannerv1alpha1.Spanne
 		log.V(0).Info("the desired number of nodes is equal to that of the current; no need to scale nodes")
 		return false
 
+	case desiredNodes > *sa.Status.CurrentNodes && r.clock.Now().Before(sa.Status.LastScaleTime.Time.Add(10*time.Second)):
+		log.Info("too short to scale up since instance scaled nodes last",
+			"now", r.clock.Now().String(),
```

Comment:
I think we should not take time with `r.clock.Now()` in here since it might slip against line 226 https://github.com/mercari/spanner-autoscaler/blob/33483fa9f900569fd2dbccff316aa22abf974b42/pkg/controllers/spannerautoscaler_controller.go#L226